.cbmc-batch/jobs/aws_array_list_swap_contents/aws_array_list_swap_contents_harness.c

```diff
@@ -0,0 +1,16 @@
+#include <aws/common/array_list.h>
+#include <proof_helpers.h>
+
+void aws_array_list_swap_contents_harness() {
+    struct aws_array_list *from;
+    ASSUME_BOUNDED_ARRAY_LIST(from, MAX_INITIAL_ITEM_ALLOCATION, MAX_ITEM_SIZE);
+
+    struct aws_array_list *to;
+    ASSUME_BOUNDED_ARRAY_LIST(to, MAX_INITIAL_ITEM_ALLOCATION, MAX_ITEM_SIZE);
+
+    __CPROVER_assume(from->alloc != NULL);
```

Comment:
Why assume this? does the code barf if its a static array? Shouldn't it check and error

Reply:
Because the code has this assertion `assert(list_a->alloc);`